unicore/distribute/api/repo_status.py

```diff
@@ -0,0 +1,54 @@
+import os
+from cornice.resource import resource, view
+from unicore.distribute.utils import (
+    get_config, format_repo_status, get_repository,
+    get_repositories, get_repository_diff, pull_repository_files)
+
+
+@resource(collection_path='status.json', path='/status/{name}.json')
```

Comment:
I wonder if a collection resource for status is useful. I'm tending towards leaving it out for the sake of simplicity.